server/src/main/java/com/alibaba/fescar/server/AbstractTCInboundHandler.java

```diff
@@ -154,12 +154,11 @@ public void execute(BranchReportRequest request, BranchReportResponse response)
      * Do branch report.
      *
      * @param request    the request
-     * @param response   the response
      * @param rpcContext the rpc context
      * @throws TransactionException the transaction exception
      */
-    protected abstract void doBranchReport(BranchReportRequest request, BranchReportResponse response,
```

Comment:
all doXXX methods here have the same param-pattern: request + response + rpcContext

doBranchReport does not need response by now, the response param is surely can be removed. But I prefer to hold it there the keep the param-pattern with others. 

Just a little suggestion. It's Okey to keep the change. :)